eslint.config.js

```diff
@@ -0,0 +1,3 @@
+'use strict'
+
+module.exports = require('neostandard')({})
```

Comment:
```suggestion
'use strict'

module.exports = require('neostandard')({
  ignores: require('neostandard').resolveIgnoresFromGitignore(),
  ts: true
})
```

Use the bog-standard one the other repos use. Needs https://github.com/fastify/gh-issues-finder/pull/415 to land though first so the .gitignore is updated. :)